### ISSUE TITLE
Possibility to remove account

### ISSUE THREAD

**Issue by Ryczko** [Ryczko/FormsLab#275]
To start working on that task please add to your `.env` file:
NEXT_PUBLIC_PROFILE_SETTINGS=true
NEXT_PUBLIC_REMOVE_ACCOUNT=true

Then you should see account page on menu, where remove account button is placed. 
![image](https://github.com/Ryczko/FormsLab/assets/51440879/551d9dec-ea6a-4c74-b040-dc9fd493ef77)
Please provide action that will remove the account **with all related surveys and data**. Please remember to secure the endpoint only for account owner.

**Comment by DevDaniloFerrari** [comment]
Hello @Ryczko, can you assign this issue to me?

**Comment by DevDaniloFerrari** [comment]
Yes, I would like to continue. I thought you had stopped the project. But I will develop this task as soon as possible.

**Comment by Ryczko** [comment]
@DevDaniloFerrari for now I'm de-assigning you and the task is up for grabs
